university-de-dschang.csl

```diff
@@ -0,0 +1,719 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
+  <info>
+    <title>University de Dschang (APA-modified)</title>
+    <title-short>UniDs</title-short>
+    <id>http://www.zotero.org/styles/university-de-dschang</id>
+    <link href="http://www.zotero.org/styles/university-de-dschang" rel="self"/>
+    <author>
+      <name>Simon Kornblith(APA)</name>
+      <email>simon@simonster.com</email>
+    </author>
+    <contributor>
+      <name>Bruce D'Arcus(APA)</name>
+    </contributor>
+    <contributor>
+      <name>Curtis M. Humphrey(APA)</name>
+    </contributor>
+    <contributor>
+      <name>Richard Karnesky(APA)</name>
+      <email>karnesky+zotero@gmail.com</email>
+      <uri>http://arc.nucapt.northwestern.edu/Richard_Karnesky</uri>
+    </contributor>
+    <contributor>
+      <name>Sebastian Karcher(APA)</name>
+    </contributor>
+    <contributor>
+      <name> Brenton M. Wiernik(APA)</name>
+      <email>zotero@wiernik.org</email>
+    </contributor>
+    <contributor>
+      <name>John Roettele (APA-modified for University de Dschang)</name>
```

Comment:
Put yourself as author, remove all the contributors.
Add <link href="http://www.zotero.org/styles/apa" rel="template"/> after line 7.

Reply:
Okay.